### PR TITLE
Use common expression on documentation regarding playbook configuration

### PR DESCRIPTION
**Pull request by luixxiul**
With some exceptions, overall the playbook uses the expression "Add the following configuration to your `inventory/host_vars/matrix.DOMAIN/vars.yml` file:" with the heading `Adjusting the playbook configuration` for sections to explain what to be added as variables.

### CODE REVIEW

File: docs/configuring-playbook-appservice-double-puppet.md [hunk 1]
```diff
@@ -6,7 +6,9 @@ This is useful for performing [double-puppeting](https://docs.mau.fi/bridges/gen
 
 Previously, bridges supported performing [double-puppeting](https://docs.mau.fi/bridges/general/double-puppeting.html) with the help of the [Shared Secret Auth password provider module](./configuring-playbook-shared-secret-auth.md), but this old and hacky solution has been superseded by this Appservice Double Puppet method.
 
-To enable the Appservice Double Puppet service, adjust your `vars.yml` configuration like this and [re-run the playbook](./installing.md) (`just install-all`):
```
Comment (luixxiul) [inline docs/configuring-playbook-appservice-double-puppet.md:9]: Please note that a dedicated section for installation by running the playbook will be created with another PR.

**Comment by spantaleev** [comment]
お疲れ様です！

Thanks! That's some extensive work unifying all of these!
